src/ensembl/index.js

```diff
@@ -183,15 +183,20 @@ const fetchAndLoadById = async (conn, { sourceId, sourceIdVersion, biotype }) =>
  */
 const uploadFile = async (opt) => {
     const HEADER = {
-        geneId: 'Gene stable ID',
-        geneIdVersion: 'Version (gene)',
+        geneIdVersion: 'Gene stable ID version',
         hgncId: 'HGNC ID',
         refseqId: 'RefSeq mRNA ID',
-        transcriptId: 'Transcript stable ID',
-        transcriptIdVersion: 'Version (transcript)',
+        transcriptIdVersion: 'Transcript stable ID version',
```

Comment:
does this mean there's now no info about which transcript is being used?

Reply:
or are transcript and geneIdVersions compounds of Id and IdVersion?

Reply:
ya they are compound (used to be separate) so we no longer need both columns. we split on '.' instead